### PR TITLE
feat: added write_to_uss and upload_file_to_uss methods

### LINKED ISSUE #37: [zos-files] Add method to upload file to USS
For this enhancement, we probably want two new methods:
- `upload_file_to_uss` modeled after `upload_file_to_dsn`:
  https://github.com/zowe/zowe-client-python-sdk/blob/4a413309c144a629bb0e87b7a6fb659a91923e30/src/zos_files/zowe/zos_files_for_zowe_sdk/files.py#L194-L201
- `write_to_uss` to call the `requests` library and write data to the USS file using the REST endpoint documented [here](https://www.ibm.com/docs/en/zos/2.4.0?topic=interface-write-data-zos-unix-file#PutWriteUnixFile)

### PR DESCRIPTION
**Pull request by himanshu007-creator**
resolves #37

### CODE REVIEW

File: src/zos_files/zowe/zos_files_for_zowe_sdk/files.py [hunk 1]
```diff
@@ -199,3 +199,28 @@ def upload_file_to_dsn(self, input_file, dataset_name, encoding=_ZOWE_FILES_DEFA
...
+
+    def write_to_uss(self, filepath_name, data, encoding=_ZOWE_FILES_DEFAULT_ENCODING):
+        """Write content to an existing UNIX file.
+        Returns
+        -------
+        json
+            A JSON containing the result of the operation
+        """
+        custom_args = self.create_custom_request_arguments()
+        custom_args["url"] = "{}fs/{}".format(self.request_endpoint, filepath_name)
```
Comment (t1m0thyj) [inline src/zos_files/zowe/zos_files_for_zowe_sdk/files.py:211]: If `filepath_name` contains a leading slash, then the string generated here will contain two consecutive slashes and the z/OSMF API doesn't like that.

I believe we could fix this by removing the leading slash if present - something like `filepath_name.lstrip("/")`

Comment (himanshu007-creator) [reply src/zos_files/zowe/zos_files_for_zowe_sdk/files.py]: thanks !

File: src/zos_files/zowe/zos_files_for_zowe_sdk/files.py [hunk 2]
```diff
@@ -199,3 +199,28 @@ def upload_file_to_dsn(self, input_file, dataset_name, encoding=_ZOWE_FILES_DEFA
...
             raise FileNotFound(input_file)
+
+    def write_to_uss(self, filepath_name, data, encoding=_ZOWE_FILES_DEFAULT_ENCODING):
+        """Write content to an existing UNIX file.
+        Returns
+        -------
+        json
+            A JSON containing the result of the operation
+        """
+        custom_args = self.create_custom_request_arguments()
```
Comment (t1m0thyj) [inline src/zos_files/zowe/zos_files_for_zowe_sdk/files.py:210]: `self.create_custom_request_arguments` -> `self.__create_custom_request_arguments`

**Review by t1m0thyj — COMMENTED** [review]
@himanshu007-creator Thanks for your contribution!

I've updated the target branch of your pull request to the newly created `next` branch. This branch contains enhancements from both the `develop` and `experimental` branches, including unit and integration tests.

Could you please address the comment below, and add a test in [test_zos_files.py](https://github.com/zowe/zowe-client-python-sdk/blob/next/tests/integration/test_zos_files.py)? The test can be modeled after `test_write_to_dsn_should_be_possible`, but for USS (`write_to_uss`) instead.

**Comment by himanshu007-creator** [comment]
Thanks ! @t1m0thyj  🙂

**Pull request merged**